### PR TITLE
Use always non GL VU-Meters in case no sufficiant GL version is available

### PR DESCRIPTION
**Pull request by daschuer**
This fixes https://github.com/mixxxdj/mixxx/issues/11286 a regression since https://github.com/mixxxdj/mixxx/pull/10893

### CODE REVIEW

**Comment by JoergAtGithub** [comment]
I tested this /w and /wo --safemode for 2.3 branch and this PR. The test showed, that this VU-Meter is broken for --safemode in 2.3 branch, but not for this PR.

I noticed the following unrelated issue: If I use --safemode, OpenGL waveforms are disabled and only the non-OpenGL waveforms are listed in the preferences, as supposed to be. But none of these non-OpenGL waveforms are rendered in Safemode, if I select them, I see just a black area.
If I start Mixxx without --safemode option and select a non-OpenGL waveform, the waveforms are working.

**Comment by JoergAtGithub** [comment]
Code LGTM

File: src/skin/legacy/legacyskinparser.cpp [hunk 1]
```diff
@@ -1300,10 +1300,13 @@ QWidget* LegacySkinParser::parseSpinny(const QDomElement& node) {
 }
 
 QWidget* LegacySkinParser::parseVuMeter(const QDomElement& node) {
-    if (!CmdlineArgs::Instance().getUseVuMeterGL()) {
+    auto* pWaveformWidgetFactory = WaveformWidgetFactory::instance();
+    if (!CmdlineArgs::Instance().getUseVuMeterGL() |
+            (!pWaveformWidgetFactory->isOpenGlAvailable() &&
```
Comment (Swiftb0y) [inline src/skin/legacy/legacyskinparser.cpp:1305]: typo which results in different semantics... Would probably have been caught by `-Wconversion`
```suggestion
    if (!CmdlineArgs::Instance().getUseVuMeterGL() ||
            (!pWaveformWidgetFactory->isOpenGlAvailable() &&
```

Comment (JoergAtGithub) [reply src/skin/legacy/legacyskinparser.cpp:1305]: I will create a PR to fix this!

Comment (Swiftb0y) [reply src/skin/legacy/legacyskinparser.cpp:1305]: Thank you.

Comment (JoergAtGithub) [reply src/skin/legacy/legacyskinparser.cpp:1305]: Here it is: #11292

**Pull request merged**